src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -371,6 +372,15 @@ public async Task RemoveAccountAsync(ApplicationConfiguration appConfig, IAccoun
                         }
                         else
                         {
+                            _logger.Info("[WamBroker] Processing WAM exception");
```

Comment:
Can this exception logic be merged into WamAdapters.HandleResponse or WamAdapters.ThrowExceptionFromWamError?